scalding-beam/src/main/scala/com/twitter/scalding/beam_backend/KryoCoder.scala

```diff
@@ -0,0 +1,29 @@
+package com.twitter.scalding.beam_backend
+
+import com.esotericsoftware.kryo.io.Input
+import com.twitter.chill.{ KryoInstantiator, KryoPool }
+import com.twitter.scalding.serialization.JavaStreamEnrichments.{ RichInputStream, RichOutputStream }
+import java.io.{ InputStream, OutputStream }
+import org.apache.beam.sdk.coders.AtomicCoder
+import scala.language.implicitConversions
+
+final class KryoCoder(kryoInstantiator: KryoInstantiator) extends AtomicCoder[Any] {
```

Comment:
Let's make it `private[beam_backend]` for now? Applies to other classes around too